### PR TITLE
Removed deprecated buildSideMenu method

### PR DESCRIPTION
**Pull request by core23**
### Subject

Removed deprecated method

### To do

- [X] Add an upgrade note

### CODE REVIEW

File: UPGRADE-4.0.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ See also the [diff code](https://github.com/sonata-project/SonataAdminBundle/com
 ## Admin
 If you have implemented a custom admin, you must adapt the signature of the following new methods to match the one in `AdminInterface` again:
  * `configureActionButtons`
+ 
+The method `buildSideMenu` was removed, please use `buildTabMenu` instead.
```
Comment (Soullivaneuh) [inline UPGRADE-4.0.md]: This is not necessary. We already have a global deprecation removal note.

**Comment by Soullivaneuh** [comment]
@core23 no `trigger_error` to be removed?

**Comment by core23** [comment]
Haven't found any `trigger_error` for this.

**Comment by Soullivaneuh** [comment]
Thank you!

**Pull request merged**